docs/06-ai-check.md

```
@@ -0,0 +1,76 @@
+# 06. 판별 기록 (Pet Checks) — Spring A 소관 범위
+
+> **[정정]** 이 문서의 이전 버전은 `POST /api/v1/ai/check` 판별 로직(규칙 엔진/LLM 하이브리드)을
+> 이 Spring 리포에 직접 만드는 것으로 설계했었다. `freepets-docs/docs/02-api-design.md`(원 설계 문서)를
+> 확인한 결과 **틀린 전제였다** — 정정한다.
+
+## 실제 아키텍처 (freepets-docs/docs/02-api-design.md 기준)
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)